packages/authgear-react-native/src/eventEmitter.ts

```diff
@@ -0,0 +1,12 @@
+import {
+  NativeModules,
+  Platform,
+  NativeEventEmitter,
+  DeviceEventEmitter,
+  EventEmitter,
+} from "react-native";
+
+export default Platform.select<EventEmitter>({
+  ios: new NativeEventEmitter(NativeModules.AuthgearManager),
+  android: DeviceEventEmitter,
```

Comment:
I just wonder if we could use the same approach (NativeEventEmitter) on android as well.

Reply:
Updated in 0bf3d99, finally found that we only need to pass undefined to `NativeEventEmitter` for non ios platform.

ref: https://github.com/facebook/react-native/blob/a715c5eba7ed7a7d8da22ba833ff135392255ba5/Libraries/Linking/Linking.js#L33